infra/gcp/dns/dns.tf

```diff
@@ -126,7 +134,7 @@ module "knative_team" {
     {
       name = "google._domainkey"
       type = "TXT"
-      ttl = 300
+      ttl  = 300
```

Comment:
This looks like an accident, but it also doesn't seem super-harmful, so I'm not going to block the PR.

Reply:
it is not an accident. ran `terraform fmt`